KubeArmor/monitor/systemMonitor.go

```diff
@@ -215,6 +245,150 @@ func isIgnored(item string, ignoreList []string) bool {
 	return false
 }
 
+// InitBPFMaps Function
+func (mon *SystemMonitor) initBPFMaps() error {
+	visibilityMap, err := cle.NewMapWithOptions(
+		&cle.MapSpec{
+			Name:       "kubearmor_visibility",
+			Type:       cle.HashOfMaps,
+			KeySize:    8,
+			ValueSize:  4,
+			MaxEntries: 65535,
+			Pinning:    cle.PinByName,
+			InnerMap:   &mon.BpfVisibilityMapSpec,
+		}, cle.MapOptions{
+			PinPath: PinPath,
+		})
```

Comment:
PinPath needs to be handled via how we mount bpffs. Can you refactor the implementation in #1097 to accomodate this usecase as well?